### PR TITLE
add real life example and summary

### PR DESCRIPTION
**Pull request by MykolaGolubyev**

### CODE REVIEW

File: blog-content/articles/testing-makes-you-faster-day-one.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-date: 2019-11-25
+date: 2019-12-19
 summary: How testing makes you fast day one and not in some unknown future
```
Comment (adeeshaek) [inline blog-content/articles/testing-makes-you-faster-day-one.md]: I really like the summary!

File: blog-content/articles/testing-makes-you-faster-day-one.md [hunk 2]
```diff
@@ -10,7 +10,12 @@ summary: How testing makes you fast day one and not in some unknown future
 *"I am not writing tests right now as they are going to slow me down"* is one of the most common excuses I heard in 
 the projects I worked on.
 
-A lack of test writing experience can make you feel this way, but I think there is another, bigger reason for the perception that tests slow you down. I believe this reason is actually an untapped superpower: the power of exercising any piece of your software in `O(1)` keystrokes. 
+A lack of test writing experience can make you feel this way, but I think there is another, 
+bigger reason for the perception that tests slow you down. I believe this reason is actually an untapped superpower:
+the power of exercising any piece of your software in `O(1)` keystrokes. By `O(1)` I mean that regardless of how big 
+your application becomes you will always be one button away of seeing the impact of your changes.  
+
+The superpower of having the tightest feedback loop.
```
Comment (adeeshaek) [inline blog-content/articles/testing-makes-you-faster-day-one.md]: This is fantastic! Amazing work. 

I have one *tiny* change to ask for here.

Your summary is a claim that has two parts: The main reason you believe tests slow you down is something that is actually a huge well of unrealized potential
Part 1: This idea is one of the main reasons you may believe tests slow you down
Part 2: The idea is an unrealized superpower

You've done a fantastic job of making Part 2 clear, but Part 1 is still not clear. Why is this a main reason that people believe tests slow you down? I strongly recommend you put one line into this intro that clarifies this. Once you do so, the thesis will become crystal clear!

File: blog-content/articles/testing-makes-you-faster-day-one.md [hunk 3]
```diff
@@ -155,5 +160,83 @@ class ChestTest {
...
+an ever growing system of teleports will make you consistently productive on day one, day two, day hundred and day thousand. 
+
+# Real Life Boring Example
+
+You probably don't face castles and dragons every day at your work, so let me map it to a boring real life example.
+
+Let's add a page to Enterprise Corporate Website. Page will list employees that have birthdays today. List must be in an alphabetic order.
+
+Your fire up your IDE, write new page, deploy to DEV environment and test it. All looks good.
+A minute later you realize that your company doesn't have employees with the same last name that have birthdays on the sae day.
```
Comment (adeeshaek) [inline blog-content/articles/testing-makes-you-faster-day-one.md]: sae - typo :)

File: blog-content/articles/testing-makes-you-faster-day-one.md [hunk 4]
```diff
@@ -155,5 +160,83 @@ class ChestTest {
 }
 ``` 
 
-This is the untapped power of tests I was talking about at the beginning - the power to exercise any piece of your software in `O(1)` keystrokes.
-Regardless of how large or small you software is, tweaking your castle and validating it in a matters of seconds using an ever growing system of teleports will make you consistently productive on day one, day two, day hundred and day thousand. 
+This is the untapped power of tests I was talking about at the beginning - the power to exercise any piece of your software in one button press.
+Regardless of how large or small you software is, tweaking your castle and validating it in a matters of seconds using 
+an ever growing system of teleports will make you consistently productive on day one, day two, day hundred and day thousand. 
+
+# Real Life Boring Example
```
Comment (adeeshaek) [inline blog-content/articles/testing-makes-you-faster-day-one.md]: This boring example is night and day. It precisely describes what a teleport is and why it helps. With this change (even without the change in the summary I recommended above) this article is ready to be published! Very impressive.

File: blog-content/articles/testing-makes-you-faster-day-one.md [hunk 5]
```diff
@@ -155,5 +160,83 @@ class ChestTest {
...
+    }
+}
+``` 
+
+Whether you implement `EmployeesSorterTest` first or `EmployeesSorter` is not that important to me.
+
+What important is you have a very tight feedback loop between your changes and the result. 
+You bypass database admins (ahem, dragon, ahem, JK), web pages, internal auth systems and get straight to the business logic at hand.
+
+# Summary
```
Comment (adeeshaek) [inline blog-content/articles/testing-makes-you-faster-day-one.md]: I really love the summary. This article does the following 3 things very well: 
1. It tells the reader what they are going to read
2. The thing
3. It tells the reader what they just read

Adding #3 by creating a summary just supercharged the article :)

Comment (MykolaGolubyev) [reply blog-content/articles/testing-makes-you-faster-day-one.md]: you are too kind, sir

File: blog-content/articles/testing-makes-you-faster-day-one.md (hunk 2 above)
Comment (MykolaGolubyev) [reply blog-content/articles/testing-makes-you-faster-day-one.md]: This makes sense. Could be a good content for a next post.

**Pull request merged**